### PR TITLE
Order menu items in a way that makes sense

### PR DESCRIPTION
**Pull request by stefwalter**
Once the redesign lands, this will probably change, but for now
in the main menu we order things by:

 * System info
 * Logs for troubleshooting
 * Configuring major subsystems: Storage, Networking
 * Stuff running on the system: Containers and Virtual Machines
 * Implementation details: Admin accounts, Services/Units

In the tools menu we just order things alphabetically.

### CODE REVIEW

File: pkg/machines/manifest.json.in [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "menu": {
      "vms": {
         "label": "Virtual Machines",
-        "path": "index.html"
+        "path": "index.html",
+	"order": 50
```
Comment (dperpeet) [inline pkg/machines/manifest.json.in]: whitespace?

File: pkg/users/manifest.json.in [hunk 2]
```diff
@@ -4,9 +4,10 @@
 	"cockpit": "122"
     },
 
-    "tools": {
+    "menu": {
         "index": {
-            "label": "Accounts"
+            "label": "Accounts",
+	    "order": 60
```
Comment (dperpeet) [inline pkg/users/manifest.json.in]: whitespace?

File: pkg/systemd/manifest.json.in [hunk 3]
```diff
@@ -9,15 +9,15 @@
...
         },
         "services": {
             "label": "Services",
-            "order": 5
+            "order": 100
         },
         "logs": {
             "label": "Logs",
-            "order": 8
+            "order": 20
```
Comment (dperpeet) [inline pkg/systemd/manifest.json.in]: logs same as storage?

File: pkg/selinux/manifest.json.in [hunk 4]
```diff
@@ -6,8 +6,7 @@
 
     "tools": {
         "setroubleshoot": {
-            "label": "SELinux",
-            "order": 8
+            "label": "SELinux"
```
Comment (dperpeet) [inline pkg/selinux/manifest.json.in]: Does a trailing comma matter here to have fewer changes? Nitpick though, doesn't really make a difference.

File: pkg/machines/manifest.json.in (hunk 1 above)
Comment (stefwalter) [inline pkg/machines/manifest.json.in]: Fixed.

File: pkg/users/manifest.json.in (hunk 2 above)
Comment (stefwalter) [inline pkg/users/manifest.json.in]: Fixed.

File: pkg/systemd/manifest.json.in (hunk 3 above)
Comment (stefwalter) [inline pkg/systemd/manifest.json.in]: Fixed and renumbered.

File: pkg/selinux/manifest.json.in (hunk 4 above)
Comment (stefwalter) [inline pkg/selinux/manifest.json.in]: JSON is unfortunately very picky about trailing commas.

**Comment by stefwalter** [comment]
> Some comments. In general I don't mind the reordering. If this becomes official policy, we should update the documentation, but I wouldn't insist on it.

Lets do that after implementing a newer design from @andreasn and @garrett

File: pkg/dashboard/manifest.json.in [hunk 5]
```diff
@@ -7,7 +7,7 @@
     "dashboard": {
         "index": {
             "label": "Dashboard",
-            "order": 0,
+            "order": 10,
```
Comment (dperpeet) [inline pkg/dashboard/manifest.json.in]: On my system, with the registry installed, Cockpit defaults to `https://[ip]:9090/kubernetes/registry` after login. Before, it defaulted to the dashboard. I guess that's related to this line.

Comment (dperpeet) [inline pkg/dashboard/manifest.json.in]: This is also why the selenium tests seem to be failing, I think.

Comment (stefwalter) [inline pkg/dashboard/manifest.json.in]: Thanks fixed.

**Comment by dperpeet** [comment]
Screenshot: 
screenshot from 2017-03-23 14-43-39

**Comment by garrett** [comment]
The ordering logic looks good to me, as does the implementation. I'm happy with everything that is shown above.

(I would like to see a non-dev screenshot if possible… but it's not really necessary.)

**Comment by andreasn** [comment]
Looks good to me!

**Pull request closed without merging**